### PR TITLE
Use ele's json to build element information

### LINKED ISSUE #435: Use ele for handling Elements
It might be nice to handle `Elements` in GMSO using the [`ele`](https://github.com/rsdefever/ele) package.

### PR DESCRIPTION
**Pull request by umesh-timalsina**
Closes #435.

### CODE REVIEW

File: gmso/core/element.py [hunk 1]
```diff
@@ -91,9 +97,9 @@ def element_by_name(name):
         Return an element from the periodic table if the name is found,
         otherwise return None
     """
-    name_trimmed = sub(r'[0-9 -]', '', name).lower()
-    msg = '''Numbers and spaces are not considered when searching by element name. \n
-             {} became {}'.format(name, name_trimmed)'''
+    name_trimmed = sub(r"[0-9 -]", "", name).lower()
+    msg = """Numbers and spaces are not considered when searching by element name. \n
+             {} became {}'.format(name, name_trimmed)"""
```
Comment (justinGilmer) [inline gmso/core/element.py:102]: Is this lone `'` on line 102 meant to be there?

File: gmso/core/element.py [hunk 2]
```diff
@@ -118,17 +124,22 @@ def element_by_atomic_number(atomic_number):
...
     """
     if isinstance(atomic_number, str):
-        atomic_number_trimmed = int(sub('[a-z -]', '', atomic_number.lower()).lstrip('0'))
-        msg = '''Letters and spaces are not considered when searching by element atomic number. \n
-                 {} became {}'.format(atomic_number, atomic_number_trimmed)'''
+        atomic_number_trimmed = int(
+            sub("[a-z -]", "", atomic_number.lower()).lstrip("0")
+        )
+        msg = """Letters and spaces are not considered when searching by element atomic number. \n
+                 {} became {}'.format(atomic_number, atomic_number_trimmed)"""
```
Comment (justinGilmer) [inline gmso/core/element.py:131]: same comment as above

File: gmso/core/element.py [hunk 3]
```diff
@@ -170,8 +181,8 @@ def element_by_mass(mass, exact=True):
         matched_element = mass_dict.get(mass_trimmed)
     else:
         # Closest match mode
-        mass_closest = min(mass_dict.keys(), key=lambda k: abs(k-mass_trimmed))
-        msg2 = 'Closest mass to {}: {}'.format(mass_trimmed, mass_closest)
+        mass_closest = min(mass_dict.keys(), key=lambda k: abs(k - mass_trimmed))
```
Comment (justinGilmer) [inline gmso/core/element.py:184]: Might be worthwhile to have a small note here or elsewhere about min, that it will only return the first minimal item.

**Review by justinGilmer — APPROVED** [review]
LGTM, but two small comments about a lone `'` i found. Other than that, its good to go!

File: gmso/core/element.py (hunk 1 above)
Comment (umesh-timalsina) [reply gmso/core/element.py]: Fixed to use format strings from Python.

File: gmso/core/element.py (hunk 2 above)
Comment (umesh-timalsina) [reply gmso/core/element.py]: Fixed to use format strings from Python.

**Review by daico007 — APPROVED** [review]
LGTM

**Pull request merged**